foreshadow/tests/test_core/test_fs_logging.py

```diff
@@ -0,0 +1,221 @@
+from unittest import mock
+
+import pytest
+
+
+@pytest.mark.parametrize("n_gets", [1, 2, 100])
+def test_get_logger(n_gets):
+    """Test the correct functionality of get_logger.
+
+    Args:
+        n_gets: number of times to try getting a logger.
+
+    """
+    from foreshadow.core import fs_logging
```

Comment:
Looking at it now, it might be worth renaming our logging module to simply [logging](https://www.tensorflow.org/api_docs/python/tf/logging)

Reply:
Will do